.eslintrc.cjs

```diff
@@ -14,6 +15,7 @@ module.exports = {
     'import/no-named-as-default-member': 0,
     'import/order': 'error',
     'import/no-extraneous-dependencies': 'error',
+    'react-refresh/only-export-components': ['error', { allowConstantExport: true }],
```

Comment:
We now have an eslint rule enforcing proper setup of react components so that hot reloading will work 
https://github.com/vitejs/vite-plugin-react-swc?tab=readme-ov-file#consistent-components-exports